vscode/oso/server/src/server.ts

```diff
@@ -11,7 +11,9 @@ const connection = createConnection(ProposedFeatures.all);
 
 const sendDiagnosticsCallback = (params: PublishDiagnosticsParams) =>
   connection.sendDiagnostics(params);
-const pls = new PolarLanguageServer(sendDiagnosticsCallback);
+const telemetryCallback = (event: unknown) =>
```

Comment:
Any more specific type here?

Reply:
Not in any way that would be helpful; the `event` param is any "JSON serializable object"

https://github.com/microsoft/vscode-languageserver-node/blob/1fe84c5b656d9db07f812cb573d2c0d82f9d8366/server/src/common/server.ts#L898-L900